app/main/views/send.py

```diff
@@ -80,6 +80,9 @@ def get_example_csv_rows(template, use_example_as_example=True, submitted_fields
 @main.route("/services/<uuid:service_id>/send/<uuid:template_id>/csv", methods=['GET', 'POST'])
 @user_has_permissions('send_messages', restrict_admin_usage=True)
 def send_messages(service_id, template_id):
+    notification_count = service_api_client.get_notification_count(service_id)
+    remaining_messages = (current_service.message_limit - notification_count)
```

Comment:
I don't think there's a need for the parentheses here; in this case they're likely not hurting anything, but in other contexts what you might end up doing is creating a [`tuple`](https://docs.python.org/3/library/stdtypes.html#tuple)

Again, in this case you're okay because we're just dealing with an arithmetic operation, though the parentheses are superfluous because they're not doing anything for us.  We don't have an order of operations situation we need to be accounting for, meaning:

```python
remaining_messages = (current_service.message_limit - notification_count)
```

is the same as:

```python
remaining_messages = current_service.message_limit - notification_count
```

...which is more readable in this case. :-)

I bring this up though because when I initially saw the parentheses they gave me pause because I couldn't remember if that'd create a tuple or not.  In case you're not familiar with tuples, they are like Python lists but with two important distinctions:

- The syntax uses `(` and `)` instead of `[` and `]`
- Tuples are immutable

Here are a couple of quick examples of tuples:

```python
# Creating an empty tuple.
empty_tuple = ()

# Creating a tuple with some elements in it.  Note the trailing comma (which is also done with regular lists).
new_tuple = ('here', 'is', 'a', 'new', 'tuple',)

# Converting a sequence (e.g., a list) to a tuple.
new_list = ['here', 'is', 'a', 'sequence',]
converted_tuple = tuple(new_list)
```

In any case, all that to say, we're okay without the parentheses here!

```suggestion
    remaining_messages = current_service.message_limit - notification_count
```